### ISSUE TITLE
Force TenantId to be Required

### ISSUE THREAD

**Issue by amerryma** [craftup/node-mongo-tenant#28]
Is there a way to force the tenantId to be required? I would like for the tenant id to never be null and I'm having trouble altering the schema after the plugin has ran. If I edit the mongo-tenant code, and add `required: true` it works. Is there something I'm missing?

**Comment by alrik** [comment]
Hey @amerryma thanks for your feature request.
I'm planning to introduce a new config option `bool requireTenantId, default false`

```js
const
  mongoose = require('mongoose'),
  mongoTenant = require('mongo-tenant');

let MySchema = new mongoose.Schema({});
MySchema.plugin(mongoTenant, { requireTenantId: true });
```
What do you think?

**Comment by amerryma** [comment]
Yes that would work nicely!

**Comment by alrik** [comment]
Hey @amerryma,

I just released version 1.3.0, including your requested feature :tada:
